lib/membership/member.js

```diff
@@ -226,4 +226,20 @@ Member.Status = {
     suspect: 'suspect'
 };
 
+/**
```

Comment:
I prefer if we keep using the same commenting style as in the rest of the project